packages/bigtable/system-test/bigtable.js

```diff
@@ -206,13 +376,12 @@ function generateTableName() {
         union: true
       };
 
-      FAMILY.setMetadata({ rule: rule }, function(err, metadata_) {
+      FAMILY.setMetadata({ rule: rule }, function(err, metadata) {
         assert.ifError(err);
+        var maxAge = metadata.gcRule.maxAge;
 
-        var maxAge_ = metadata_.gcRule.maxAge;
-
-        assert.equal(maxAge_.seconds, rule.age.seconds);
-        assert.strictEqual(maxAge_.nanas, rule.age.nanas);
+        assert.equal(maxAge.seconds, rule.age.seconds);
```

Comment:
So actually I discovered that the reason for this is that the upstream API converts it to a string and does not return it as a number.